### PR TITLE
Userinfo content type test updates

### PR DESCRIPTION
**Pull request by c00crane**
Update the userinfo tests that verify that we don't accept a jwt when the content-type is json.
New messages have been added and we need to ensure that we get these messages.
Also adding tests that make sure we won't accept a json formatted response when the content-type is jwt.

### CODE REVIEW

**Comment by skasund** [comment]
L2 message review completed

**Comment by dmuelle** [comment]
I don't see any messages here that need ID review- if that's not correct, let me know. Thanks

**Comment by c00crane** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.jwt_fat.builder,com.ibm.ws.security.jwt_fat.consumer,com.ibm.ws.security.jwtsso_fat,com.ibm.ws.security.mp.jwt.1.1_fat.commonTest,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.1,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-2.0,com.ibm.ws.security.mp.jwt.1.2_fat.commonTest,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-2.0,com.ibm.ws.security.oauth_fat,com.ibm.ws.security.oidc.client_fat.1,com.ibm.ws.security.oidc.client_fat.2,com.ibm.ws.security.oidc.client_fat.jaxrs,com.ibm.ws.security.oidc.client_fat.spnego,com.ibm.ws.security.oidc.server_fat,com.ibm.ws.security.oidc.server_fat.jaxrs.config,com.ibm.ws.security.oidc.server_fat.oidc,com.ibm.ws.security.oidc.server_fat.spnego,com.ibm.ws.security.social_fat,com.ibm.ws.security.social_fat.LibertyOP,com.ibm.ws.security.spnego_fat,com.ibm.ws.security.spnego_fat.1,com.ibm.ws.security.spnego_fat.2

**Comment by c00crane** [comment]
#run-libbybot
#libby

**Comment by c00crane** [comment]
@OpenLiberty/message-reviewer the changes in this pull may have already been reviewed with pull #19097

File: dev/com.ibm.ws.security.oidc.client_fat.2/bnd.bnd [hunk 1]
```diff
@@ -20,6 +20,13 @@ publish.wlp.jar.disabled: true
 tested.features: jsp-2.3, el-3.0, restfulwsclient-3.0, restfulws-3.0,\
     appsecurity-4.0, expressionlanguage-4.0, pages-3.0
 
+Import-Package: \
+    !*.internal.*, \
+    !com.ibm.ws.kernel.boot.cmdline, \
+    org.apache.commons.codec.binary,\
+    org.jose4j.*, \
+    *
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.client_fat.2/bnd.bnd:28]: I'm surprised these lines would be needed with the `*` import there.

**Review by ayoho — APPROVED** [review]

**Review by teddyjtorres — APPROVED** [review]

**Pull request merged**